tests/test_poincare_ball.py

```diff
@@ -237,3 +237,18 @@ def test_ball_retraction(self):
         ball_metric = self.manifold.metric
         tangent_vec = ball_metric.log(y, x)
         ball_metric.retraction(tangent_vec, x)
+
+    def test_ball_geodesic(self):
+        path_function =\
+            self.manifold.metric.geodesic(gs.array([0.1, 0.1]),
+                                          gs.array([0.2, 0.2]))
+        steps = gs.array(gs.linspace(-1000., 1000., 10000))
+        path_function(steps)
+
+    def test_mobius_out_of_the_ball(self):
```

Comment:
To check that a method raises the error that is expected, use:
`assertRaises(ValueError)`
e.g. 

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises